utils/chunk-tests.js

```diff
@@ -16,6 +16,28 @@ const runnersMap = new Map([
   ['test-dev', { min: 1, max: 5, runners: ['ubuntu-latest', 'macos-latest'] }],
 ]);
 
+const optionsOverrides = {
+  examples: { min: 1, max: 1 },
+};
+
+function getRunnerOptions(scriptName, packageName) {
+  let runnerOptions = runnersMap.get(scriptName);
+  if (optionsOverrides[packageName]) {
+    runnerOptions = Object.assign(
+      {},
+      runnerOptions,
+      optionsOverrides[packageName]
+    );
+  }
+  return (
+    runnerOptions || {
+      min: 1,
+      max: 1,
```

Comment:
I suggest using `max: 2` or perhaps `max: 3` here

Reply:
Ack. I thought I was testing this uncached, but I must have been looking at cached runs. Will update.

Reply:
PR: https://github.com/vercel/next.js/pull/47980